components/Profile/ExerciseChart.jsx

```diff
@@ -90,7 +89,7 @@ export default () => {
   return (
     <ExerciseChartContainer>
       <TwoColumnView>
-        <ExerciseName>{exerciseState.selectedExerciseName || 'Select an Exercise'}</ExerciseName>
+        <ExerciseName longText={exerciseState.selectedExerciseName && exerciseState.selectedExerciseName.length >= 20}>{exerciseState.selectedExerciseName || 'Select an Exercise'}</ExerciseName>
```

Comment:
This fix should work most of the time, but in the case where they create a custom exercise with a really long name, we could run into the same issue. Should we cap the custom exercise name length at a certain amount of letters to prevent this from happening.

Reply:
Added 25 character limit to custom exercise names